Hirsch_sites/EGRET/EGRETtoANA.R

```diff
@@ -0,0 +1,90 @@
+EGRETtoANA <- function(egret.files, ANA.dir) {
+  
+  library(dplyr)
+  library(EGRET)
+  library(loadflex)
+  
+  # read the RData files and confirm that they're EGRET eLists
+  elists <- lapply(egret.files, function(egret.file) {
+    elist.name <- load(egret.file)
+    egret.elist <- get(elist.name)
+    if(!all.equal(names(egret.elist), c('INFO','Daily','Sample','surfaces')) || class(egret.elist) != 'egret') {
+      stop("are you sure this file contains an eList?")
+    }
+    return(egret.elist)
+  })
+  
+  # extract the INFOs into a siteInfo table and write to csv
+  constitInfo <- bind_rows(
+    lapply(elists, function(elist) {
+      elist$INFO %>%
+        mutate(
+          conc.units = {
+            if(all(grepl('mg/l', tolower(param.units)))) {
+              'mg L^-1'
+            } else {
+              stop('expected param.units of mg/l, per Table 2 of https://pubs.usgs.gov/tm/04/a10/pdf/tm4A10.pdf')
+            }
+          },
+          matching.site = staAbbrev
+        ) %>%
+        select(
+          matching.site,
+          site.id = staAbbrev, # as in convertToEGRET
+          site.name = shortName, # as in convertToEGRET
+          lat = dec_lat_va,
+          lon = dec_long_va,
+          basin.area = drainSqKm,
+          constituent = constitAbbrev, # as in convertToEGRET
+          consti.name = paramShortName, # as in convertToEGRET
+          units = conc.units # as in convertToEGRET. always mg/L
```

Comment:
"as in convertToEGRET" meaning that what's on the right side is output once that function is used?